### DISCUSSION TITLE
PostProcessing Sprite sometimes not visible on the screen

### DISCUSSION THREAD

**Discussion by goldenratio** [jnsmalm/pixi3d#141] [Q&A]
Hey @jnsmalm,
Sorry for this weird question, but I would like to hear your thoughts.

We have couple of models, that goes through PostProcessing sprite and gets a outline filter. Super randomly, only those PostProcessing sprite doesn't show up on the screen (most of the time it shows up), other models visible all the time.

I have tried disabled Pixi's GC, but it still happens.

Have you faced something like this before?  Your help is really appreciated on this.
please note: Code example doesn't help, as it happens super randomly.

**Comment by jnsmalm** [comment]
Are the models visible at first and later disappear, or they don't show up at all from the start when this happens? Did you try to use a tool like Spector.js? If you use that tool you might get more clues, maybe they render but later the texture gets overwritten or something? Does it happen on only certain devices?

**Reply by jnsmalm** [reply]
Have not seen this before and I have used PostProcessingSprite in production running on very many different devices.

**Reply by goldenratio** [reply]
thanks, I will keep debugging further. It happens from the start and in all devices but super randomly, once in 25ish times or so
